### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by chaudharykeshavraj** [community/community#64291] [GitHub Education] [answered]
How much time does it take to recieve student developer pack benefits after being approved?

**Comment by nirav-gajera** [comment, marked as answer]
It usually takes 1-2 business days to receive the benefits of the GitHub Student Developer Pack after being approved. However, it may take longer depending on the volume of applications that GitHub is receiving.

**Reply by sksaleha** [reply]
Hey i have a dought my application was sucessfully verified but it shows like these:  Congrats! Your request for [@sksaleha](https://github.com/sksaleha) was approved and your benefits will be available soon. 
what does it mean can you help me?

**Reply by nirav-gajera** [reply]
it means that your application for the GitHub Student Developer Pack has been approved. However, the benefits will not be available immediately. It usually takes 3-5 business days for the benefits to be activated after your application is approved.

In your case, your application was approved on August 27, 2023. So, the benefits should be available to you by September 1, 2023 at the latest. However, it is possible that the benefits may be available sooner

**Reply by sksaleha** [reply]
thanks! alot brother

**Reply by sksaleha** [reply]
Bro I have another dought is there any other chances to get reject of my student developer pack as my current status is : Congrats! Your request for @sksaleha was approved and your benefits will be available soon.

**Reply by nirav-gajera** [reply]
no after accepted there is no chance to reject only benefits maybe take time sometimes

**Reply by swwapnilrg** [reply]
@nirav-gajera my account shows this (Congrats! Your request for [@swwapnilrg](https://github.com/swwapnilrg) was approved and your benefits are now available.)
but  i cant see any benefits  activated.i dont understand i submitted on 25th aug got approved on 28 and it says (the benefits are now available).
[](url)
<img width="338" alt="Screenshot 2023-08-30 at 3 12 12 PM" src="https://github.com/community/community/assets/115607505/53d8f67b-5a5b-4e70-a501-e376e681c48b">

**Reply by thiagohora** [reply]
I have the same issue. It says approved, but I cannot use partner services like Datadog: "Student verification failed. There was a problem verifying your student status. [Please contact Github](https://github.com/contact) to confirm your status or enrol."

**Reply by FoxtrotCN** [reply]
I have exactly the same issue but trying to link the account with JetBrains. I don't understand what's the problem

**Reply by ultard** [reply]
@swwapnilrg Lucky you, it won't even let me submit an application when I've filled out everything correctly.

![image](https://github.com/community/community/assets/63789651/c7a818a7-5d59-4939-bbd1-31432dbf77c2)
![image](https://github.com/community/community/assets/63789651/1652842f-f9ba-41de-abff-e931398f3b17)

**Reply by isjunrod** [reply]
My application was accepted on August 19: Congratulations! Your application for @JunRod was approved and your benefits are now available. But until today they don't give me the benefits of Github students :c.
![Screenshot_1](https://github.com/community/community/assets/87834204/70bae3a6-e65f-427b-9ffa-544b5e7369cd)
![Screenshot_2](https://github.com/community/community/assets/87834204/ab496579-d1e7-43b6-b0fc-0608fe48ef50)

**Reply by Sino-cant-code** [reply]
Hiii, I am facing the same problem. It say's it is aproved but no access . Can you tell me how many days it took in your case?

**Reply by serhankoral** [reply]
![image](https://github.com/user-attachments/assets/91cca20e-f0e2-44ae-b42f-2436c7f10c65)

My application was approved on 31.12.2024, but the Student Developer Package gift has not been issued yet

**Comment by Herand19** [comment]
> It usually takes 1-2 business days to receive the benefits of the GitHub Student Developer Pack after being approved. However, it may take longer depending on the volume of applications that GitHub is receiving.

**Comment by jzhang3390** [comment]
Hi I try to apply for the Student Developer Pack but one reason was got rejected due to I am a online learner and the location is far from my campus. How to solve this issue?

**Reply by aholcomb23** [reply]
I am also enrolled at a university and the provided courses are fully online. I took a picture of my acceptance letter that has the university letterhead, a statement saying that it's a fully online program, the date of my acceptance, and the dates of the upcoming coursework. What else can I provide to get the student developer pack? Please help. Thank you!

**Comment by Jwool23** [comment]
Is there a limited amount of times you can submit proof that you go to your school?

**Reply by AnujPandey007** [reply]
Only while you are a student you can do

**Comment by lucy2512** [comment]
I was using the github student developer pack for last 2 years, but still I am a student and I want to re-verify my student id and want to get the student developer pack benefits again. It is very much important for my projects. Previously I have requested for 9 times but every it got rejected now I am not able to upload or request further please help me. It is showing that you have outstanding error.
Please help me

**Reply by da-b0ss** [reply]
Im getting the same problem, i submitted incorrectly the first time and they denied my request, ive sent multiple new requests and they just havent done anything/reviewed them, its been like that for a couple weeks now. Did you end up fixing the problem?

**Comment by govindms777** [comment]
My request was approved 3 days ago on september 1st however I still haven't received the benefits yet. Is there anything I could do to speed up the process as I was hoping to use the services in the pack for an academic project which I have to complete within a week.

**Reply by admztech** [reply]
have you received your benefits or not ????

**Comment by guvenaltunsoyy** [comment]
I'm waiting since 4 September. Is there any news?

**Reply by swwapnilrg** [reply]
It'd take 3 more days Max to max

**Comment by ruhankhandakar** [comment]
It's been 10 days still not able to access the benefits (31st Oct they have approved)

**Comment by AbsurdumDaemonium** [comment]
It's literally been like 2 months

**Comment by GordonSlyding** [comment]
having same issue, any updates?

**Comment by dasxunya** [comment]
I have the same problem, my request was approved since 21 November, but benefits aren’t available
I really need it for studying, is it possible to speed up the process?

**Comment by mscode07** [comment]
I applied for it but no confirmation I got after giving the location access to it come back to the form page what to do?

**Reply by AbsurdumDaemonium** [reply]
it's like this for me too, but it tells me they are reviewing it

**Reply by mscode07** [reply]
How much time for reviewing they take?

**Comment by niceEli** [comment]
i have applied in sept and still don't have befits. it just says we are reviewing your application at the top

**Reply by FifthWit** [reply]
I'd reapply and try to more closely follow guidelines, for me it was automatically approved and I just have to wait for the benefits to arrive

**Reply by niceEli** [reply]
> I'd reapply and try to more closely follow guidelines, for me it was automatically approved and I just have to wait for the benefits to arrive

I'll try and do that I guess because I already waited 5 months so what's the point of not trying again

**Reply by brettppark** [reply]
i reapplied several times but processing information button doesn't work.
![image](https://github.com/community/community/assets/157556665/3c649000-6c77-4e69-862c-1da79ebfc1ea)

**Comment by J1mb0t** [comment]
I've applied for the student developer pack a few months ago it still says that "We're still processing your application. Once this is complete you will have access to the student developer pack offers."
I just wanted to see what's going on and or/if you had forgotten me?

**Reply by niceEli** [reply]
It still says that for me aswell

**Reply by KlausJackson** [reply]
Same.

**Comment by sinister-virus** [comment]
I have not received any benefits in the last 10 months.
The approved is Purple, but why it hasn’t turned green yet?
![Screenshot 2024-04-26 154421](https://github.com/community/community/assets/51523645/b095a030-8cf1-4b4a-b59d-ecb96725a3fd)
![Screenshot 2024-04-26 154434](https://github.com/community/community/assets/51523645/592a1319-761d-488e-9750-163384e8dbb3)

**Comment by Tanay2920003** [comment]
It happens maybe due to some fault

**Comment by MACHAVARAMLAKSHMISAMRAJYAANANYA** [comment]
I HAVE APPLIED FOR STUDENT DEVELOPER PACK AND IT ALREADY BEEN 10 DAYS WHEN WILL I GET THE ACCESS .PLS REPSOND TO IT

**Comment by codewitheren** [comment]
when can github developer student pack gifts be received?

**Comment by auspicious27** [comment]
<img width="1440" alt="Screenshot 2024-09-17 at 9 01 47 AM" src="https://github.com/user-attachments/assets/4da1f4a6-a9b1-4b1a-8d52-b09c8e254ebc">
I am unable to submit . kindly help me

**Comment by josephmorris423** [comment]
It' been 10 days and my application is approved but benefits are not granted.

**Reply by KlausJackson** [reply]
Wait for a little longer.

**Comment by Himani0002** [comment]
Hello

**Reply by Himani0002** [reply]
Hii

**Comment by samus-aran** [comment]
We're locking this discussion because it has not had recent activity and/or there are more recent discussions that answers more efficiently. Thank you for helping us maintain a productive and tidy community for all our members.
